packages/@n8n/db/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@n8n/db",
-  "version": "0.2.0",
+  "version": "0.1.0",
```

Comment:
Is this to fix a wrong copy paste ?

Reply:
Changed this thinking it was a copypaste mistake, but it looks like we have a GH action updating these automatically, so I'll revert.